chandra_aca/star_probs.py

```diff
@@ -497,3 +499,53 @@ def prob_minus_p_acq(mag):
         mag = float(mag)
 
     return mag
+
+
+def guide_count(mags, t_ccd):
+    """
+    Given mags from guide stars and a temperature, calculate a guide star
+    fractional count/metric using signal-to-noise scaled mag thresholds.
+
+    :param mags: mags of guide star catalog stars
+    :param t_ccd: ACA CCD temperature at expected time of observation
+    :returns: fractional count
+    """
+    # The bright limit does not scale.
+    thresh0 = 5.9
+    thresh1 = snr_mag_for_t_ccd(t_ccd, ref_mag=10.0, ref_t_ccd=-10.9)
```

Comment:
Though now that I'm implementing in chandra_aca, I wonder if we would be better served by having chandra_aca.transform.snr_mag_for_t_ccd have those default kwargs, mostly to keep the ref_t_ccd as an actual reference, instead of that living here.

But `guide_count` does have a lot of other pieces that are somewhat characteristics anyway.

Reply:
Yes, *every* number in this function is a characteristic of the SSAWG-approved algorithm for counting guide stars.  This is effectively the reference implementation, so I think it makes complete sense to have them all specified locally here in the code.  One could factor them out as named constants, but to my mind that would only reduce readability.

`snr_mag_for_t_ccd` is a different beast and has general use beyond just this function of counting stars.  That's why I don't want to embed a constant related to counting stars into that function.